packages/language-server/test/snapshots/xml-view-diagnostics/unknown-xmlns-namespace/diagnostic-spec.ts

```diff
@@ -0,0 +1,6 @@
+import { snapshotTestLSPDiagnostic } from "../snapshots-utils";
+describe(`The language server diagnostics capability`, () => {
```

Comment:
newline?

Reply:
I ran prettier... it doesn't have a problem with the line length

Reply:
I meant between import blocks and rest of code?

Reply:
It's the same as in the other snapshot tests. I think it's ok because it's basically a placeholder tepmlate... maybe we can even remove these files and generate the tests on the fly according to the folder names?

Reply:
I'll try to generate them and see if it looks better

Reply:
See #104